### PR TITLE
Fixes cargo, ends the memes.

### PR DESCRIPTION
**Pull request by Niknakflak**
What this PR does
- Replaces cash lemons with combustible lemons.  Whoever made lemon mutations be "cash" instead of a firebomb is dumb because the combustible lemon is legendary and a way better meme/~~reference~~
- Combustible lemons are more or less identical to firebombs with the lower potency ones being slightly weaker with the max potency ones being slightly stronger.  Can be changed if need be because its probably easier to make an IED than to grow these, but these can also be mass produced easier in later tier botany and really, botany isn't that hard.

Why this should be merged
- shuts ma44 the fuck up
- fixes cargo cash lemon meme shit 
- dank ~~reference~~ item.

I didnt test this, but it compiled.  Tear it apart for code first.

### CODE REVIEW

File: code/modules/hydroponics/grown/citrus.dm [hunk 1]
```diff
@@ -68,56 +68,82 @@
...
-	icon_state = "moneyfruit"
+/obj/item/weapon/reagent_containers/food/snacks/grown/firelemon
+	seed = /obj/item/seeds/firelemon
+	name = "Combustible Lemon"
+	desc = "Made for burning houses down."
+	icon_state = "firelemon"
 	bitesize_mod = 2
 
-/obj/item/weapon/reagent_containers/food/snacks/grown/shell/moneyfruit/add_juice()
+/obj/item/weapon/reagent_containers/food/snacks/grown/cherry_bomb/attack_self(mob/living/user)
```
Comment (Iamgoofball) [inline code/modules/hydroponics/grown/citrus.dm]: forgot to change the path

**Comment by ma44** [comment]
>explosion(src.loc,-1,-1,2, flame_range = 5)
jesus fuck you know i can easily mass produce this shit, you trying to make the station burn down?
Also it won't shut ma44 the fuck up, as i will be complaining about lack of cash lemons :(

File: code/modules/hydroponics/grown/citrus.dm (hunk 1 above)
Comment (Niknakflak) [inline code/modules/hydroponics/grown/citrus.dm]: im just so good

**Comment by Ninja137** [comment]
If it's going to take more effort, then it should probably be better than the shitty IEDs. At minimum, at least make them explode on impact, rather than have the shitty delay.

**Comment by ma44** [comment]
@Ninja137 you can literally just spend 15 minutes and easily have a hundred of these fuckers

**Comment by Ninja137** [comment]
A hundred worthless minor firebombs that have a random delay on impact don't mean a thing.

**Comment by PKPenguin321** [comment]
>opened with conflicts

this is why you update your fork before making a new branch

**Comment by Niknakflak** [comment]
I did

**Comment by Incoming5643** [comment]
If you can believe it, I think cash lemons predate portal 2

**Comment by Niknakflak** [comment]
where are they from?

**Comment by Incoming5643** [comment]
They're not from anything as far I know, I'm saying that I think they existed in the codebase before portal 2 came out.

**Comment by RemieRichards** [comment]
@Incoming5643 cash lemons aren't a portal 2 reference, you're thinking of combustible lemons from Cave Johnson's rant.

**Comment by Incoming5643** [comment]
I'm not saying they are!

I'm saying they predate portal 2 so you couldn't have used portal 2 as a competing reference for a mutated lemon.

**Comment by RemieRichards** [comment]
@Incoming5643 yes, and I'm saying CASH lemons (which is what /you/ said) have nothing to do with portal 2, which they don't!

**Comment by Incoming5643** [comment]
I'm saying that when the idea for a mutated lemon was spawned from the head of some old world coder, the date was not yet April 19, 2011.

Thus said coder could not have thought to use combustible lemons as a reference, as said reference did not yet exist.

Thus cash lemons came to exist instead.

**Comment by PKPenguin321** [comment]
@RemieRichards niknak was wondering why it was cash lemons not combustible lemons since that was the more obvious reference
@Incoming5643 was saying that when cash lemons were made combustible lemons werent the obvious reference because that reference did not exist
the status of cash lemons as a reference or not as a reference is irrelevant

**Comment by Niknakflak** [comment]
I dont understand why remie did not understand

**Comment by ma44** [comment]
https://file.house/IWn4.jpeg
me irl
*credit to iamgoofball*

**Comment by Niknakflak** [comment]
get your fuckin shitpost outta here

**Comment by KorPhaeron** [comment]
us gamers huh

**Comment by ma44** [comment]
"This branch has conflicts that must be resolved"

**Comment by Niknakflak** [comment]
Yes, I can read

**Comment by Iamgoofball** [comment]
"This branch has conflicts that must be resolved"

**Comment by Jazaen** [comment]
Can max potency combustible lemons be used to destroy rock?

**Comment by optimumtact** [comment]
niknakscrewwsthepooch

**Comment by PKPenguin321** [comment]
:raised_hands: :clap: :tada: :balloon:

**Pull request merged**